plugins/buildx.installer

```diff
@@ -18,8 +19,10 @@ build() {
         git fetch --all
         git checkout -q "${COMMIT}"
         local LDFLAGS
-        # TODO: unmark `-tp` when no longer a technical preview
-        LDFLAGS="-X ${PKG}/version.Version=$(git describe --match 'v[0-9]*' --always --tags)-tp -X ${PKG}/version.Revision=$(git rev-parse HEAD) -X ${PKG}/version.Package=${PKG}"
+        local VERSION
+        VERSION=$(git describe --match 'v[0-9]*' --always --tags)
+        test -n "${EXPERIMENTAL_PLUGIN}" && VERSION=${VERSION}-tp
```

Comment:
Wait what is this supposed to do?

Reply:
I can remove it. It just does what you had in a comment: add `-tp` if it's experimental.